package.json

```
@@ -61,10 +61,20 @@
 							},
 							"color": {
 								"type": "string",
-								"description": "The color in HEX format to apply to your workspace"
+								"description": "The background color in HEX format to apply to your workspace"
+							},
+							"foregroundColor": {
+								"type": "string",
+								"description": "The foreground color for the status bar and/or activity bar. It is composed by a Regular Expression (regex) to identify the org alias and the color to apply in HEX format."
```

@jlawrence124 to keep it consistent with the background attribute, I suggest to have a similar description (I don't think the regex is applying to the foregroundColor HEX format, but only to the rules)
```suggestion
								"description": "The foreground color in HEX format to apply to your workspace"
```